dev/com.ibm.ws.classloading/resources/OSGI-INF/l10n/classloader.properties

```diff
@@ -33,7 +33,7 @@ classloader.common.library.ref$Ref=Shared library references
 classloader.common.library.ref.desc=List of library references. Library class instances are shared with other classloaders.  
 # allowed api types
 classloader.apis=Allowed API types
-classloader.apis.desc=The types of API packages that this class loader supports. This value is a comma-separated list of any combination of the following API packages: spec, ibm-api, api, stable, third-party.
+classloader.apis.desc=The types of API packages that this class loader supports. This value is a comma-separated list of any combination of the following API packages: spec, ibm-api, api, stable, third-party.  If a prefix of + or - is added to API types, they will be added or removed, respectively, from the default set of API types.  Common usage for prefix "+third-party" results in "spec, ibm-api, api, stable, third-party" and "-api" results in "spec, ibm-api, stable" 
```

Comment:
Suggestion:
If a prefix of + or - is added to API types, those API types are added or removed, respectively, from the default set of API types.  Common usage for the prefix, +third-party, results in "spec, ibm-api, api, stable, third-party". The prefix, -api, results in "spec, ibm-api, stable". 

Are these API types: "spec, ibm-api, api, stable, third-party" and "spec, ibm-api, stable"

Reply:
Completed changes